client/my-sites/site-settings/test/form-general.jsx

```diff
@@ -159,7 +159,7 @@ describe( 'SiteSettingsFormGeneral ', () => {
 				testProps.fields.blog_public = initialBlogPublic;
 				const { getByLabelText } = renderWithRedux( <SiteSettingsFormGeneral { ...testProps } /> );
 
-				const radioButton = getByLabelText( text );
+				const radioButton = getByLabelText( text, { exact: false } );
```

Comment:
Why is the `{ exact: false }` added here (and below)?

Reply:
The label of the checkbox now spans html elements because of the added `<FormSettingExplanation>`. `getByLabelText`, by default, looks for exact matches and does not match text across elements: 

https://testing-library.com/docs/dom-testing-library/api-queries
> `ByLabelText`
>It will NOT find the input node for label text broken up by elements.

Reply:
I just saw it in your PR description as well 🤦 Sorry I didn't read it more carefully there. Thanks for the explanation!